apps/condo/domains/billing/schema/BillingAccount.js

```diff
@@ -65,6 +66,21 @@ const BillingAccount = new GQLListSchema('BillingAccount', {
             isRequired: false,
         },
 
+        isClosed: {
+            schemaDoc: 'Shows whether the billing account closed or not',
```

Comment:
o_O, why we just not mark it as deleted? It looks as extra complexity and we need an architecture decision and slide before

Reply:
Schemadoc improved